docs/references/ic-interface-spec.md

```diff
@@ -1390,6 +1394,17 @@ While an implementation will likely try to keep the interval between the value o
 
 :::
 
+#### On Low Wasm Memory {#on-low-wasm-memory}
+
+A canister can export a function with the name `canister_on_low_wasm_memory`, which is scheduled whenever the canister's remaining wasm memory size in bytes falls from at least a threshold `t` to strictly less than `t`.
```

Comment:
@mraszyk I do not think this is true, hook is executed whenever after memory grows, remaining wasm memory is below `t`. Two cases: 
1. The remaining wasm memory size in bytes falls from at least a threshold `t` to strictly less than `t`
2. The remaining wasm memory size was less than threshold `t`, and after growing, the remaining wasm memory additionally decreased.